oar/core/advisory.py

```diff
@@ -1,3 +1,5 @@
+from datetime import datetime
+from dateutil import parser
```

Comment:
dateutil is dependency of `rh-doozer`
https://github.com/openshift/release-tests/blob/a8e69c150d340ac72dc6282cb765020e07ca2802/pyproject.toml#L46

Reply:
Should I change something here? I am not sure what do you mean.

Reply:
nothing, just need to confirm if new package needs to be added to install profile